app/models/task.py

```diff
@@ -1,5 +1,25 @@
 from app import db
 
-
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    task_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    goal_id = db.Column(db.Integer, db.ForeignKey('goal.goal_id'))
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime)
+
+    def create_task_dict(self):
+        task_dict = {
+                "id": self.task_id,
+                "title": self.title,
+                "description": self.description
+        }
+        if self.completed_at is None:
+            task_dict["is_complete"] = False
+        else:
+            task_dict["is_complete"] = True
```

Comment:
This can be simplified to:

```suggestion
        task_dict = {
                "id": self.task_id,
                "title": self.title,
                "description": self.description,
                "is_complete": bool(self.completed_at)
        }
```